src/main/webapp/add-course.html

```diff
@@ -4,49 +4,88 @@
         <meta charset="UTF-8">
         <title>Add Course Page</title>
         <link rel="stylesheet" href="search-style.css">
-        <script src="https://apis.google.com/js/platform.js" async defer></script>
+        <link
+            rel="stylesheet"
+            href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.1/css/bootstrap.min.css"
+            integrity="sha384-VCmXjywReHh4PwowAiWNagnWcLhlEJLA5buUprzK8rxFgeH0kww/aWY76TfkUoSX"
+            crossorigin="anonymous"
+        />
         <meta name="google-signin-client_id" content="12523748853-6h3fnh1kppht9s895ist2b5222ifh2u7.apps.googleusercontent.com">
-        <script src="search-script.js"></script>
-        <script src="login-script.js"></script>
+
     </head>
     <body>
```

Comment:
give this a "class=bg-light" so you don't have a white background for the logout button", basically the whole page will be that grey color

Reply:
Thanks! Was wondering how to fix that